packages/components-js/tests/vrt/playwright/specs/common/components.vrt.ts

```diff
@@ -155,13 +155,10 @@ components.forEach((component) => {
     // print view
     baseThemes.forEach((theme) => {
       test(`should have no visual regression for printed pdf with theme ${theme}`, async ({ page }) => {
-        test.skip(
-          component === 'toast' || component === 'stepper-horizontal',
-          'Toast and stepper-horizontal are flaky'
-        );
+        const flakyPrintComponents = ['scroller', 'stepper-horizontal', 'tabs', 'tabs-bar', 'toast', 'flyout'];
```

Comment:
Now we are skipping more tests than before?

Reply:
Since we don't support print, i think it doesn't matter